Framework/Algorithms/src/LineProfile.cpp

```diff
@@ -225,7 +225,8 @@ void profile(std::vector<double> &Xs, std::vector<double> &Ys,
     for (size_t j = limits.widthStart; j < limits.widthEnd; ++j) {
       const size_t iHor = dir == LineDirection::horizontal ? i : j;
       const size_t iVert = dir == LineDirection::horizontal ? j : i;
-      auto iter = ws.histogram(iVert).begin();
```

Comment:
Interesting bug. Can we prevent this by adding an lvalue reference qualifier to `Histogram::begin()` and `end()`, just like I had done for the day-1 methods in `Histogram` such as `setBinEdges()`?

Reply:
Good idea. :+1: